code/modules/admin/topic.dm

```diff
@@ -47,6 +47,48 @@
 			to_chat(usr, "<span class='danger'>ERROR: Mob not found.</span>", confidential = TRUE)
 			return
 		cmd_show_exp_panel(M.client)
+
+// SKYRAT EDIT BEGIN -- ONE CLICK ANTAG
+	else if(href_list["makeAntag"])
+
+		message_admins("[key_name_admin(usr)] is attempting to make [href_list["makeAntag"]]")
+
+		if(!check_rights(R_ADMIN))
+			return
+
+		if (!SSticker.mode)
```

Comment:
Isn't gamemode being deleted soon?

Reply:
SSticker.mode is used to store if a round has started, and was also used to store the mode itself; all that does is check if the game has started